glean-core/rlb/src/test.rs

```diff
@@ -1284,6 +1284,36 @@ fn test_boolean_get_num_errors() {
     assert_eq!(result, 0);
 }
 
+#[test]
+fn test_labeled_counter_metric() {
+    let _lock = lock_test();
+
+    let _t = new_glean(None, false);
+
+    let metric = LabeledCounter::new(
+        LabeledMetricData::Common { cmd: CommonMetricData {
+            name: "labeled_counter".into(),
+            category: "telemetry".into(),
+            send_in_pings: vec!["store1".into()],
+            disabled: false,
+            lifetime: Lifetime::Ping,
+            ..Default::default()
+        } },
+        Some(vec!["key1".into()]),
+    );
+
+    metric.get("key1").add(1);
```

Comment:
For the other binding tests, you had two labels. Any reason you only have one here?

Reply:
Nope lol that was an accident